R/check-cols-complete.R

```diff
@@ -23,20 +23,25 @@ check_cols_complete <- function(data, required_cols,
   if (is.null(data)) {
     return(NULL)
   }
-  ## Check if columns in `required_cols` contain missing data
+  req_cols <- required_cols[required_cols %in% names(data)]
```

Comment:
It's a small thing but could we give this a more informative name? I'm already mixing up `required_cols` and `req_cols` as I'm reading the rest of the code. Maybe `req_cols_present` or something?

Reply:
For sure! I have a tendency towards overly long variable names. Restraining myself here made this section a bit confusing. Haha.